cypress/lib/helper.js

```diff
@@ -35,4 +35,37 @@ export function checkProspectsAreVisible() {
   cy.contains(prospect1)
   cy.contains("Prospect 2")
   cy.contains("Prospect 10")
+}
+
+export function checkBookingsOrder(expectingData) {
+  const NUMBER_OF_FIELDS = 50
+  for (let index = 0; index < NUMBER_OF_FIELDS; ++index) {
+    cy.get('p').eq(index).contains(expectingData[index])
+  }
+}
+
+export function createExpectingData(firstBooking, ordering, bookingsNumber) {
```

Comment:
why not hard code an array of ids and get the other booking info based on that booking id?

Reply:
I don't understand this part. how can we get all the data of a booking according to the "id"?